packages/node_modules/@webex/internal-plugin-conversation/src/decryption-transforms.js

```diff
@@ -46,6 +46,56 @@ export const transforms = toArray('inbound', {
     return ctx.transform(`decrypt${capitalize(object.objectType)}`, key, object);
   },
 
+
+  /**
+   * Decrypt an individual reaction2Summary activity (object.objectType === 'reaction2Summary')
+   * @param   {Object} ctx      An object containg a webex instance and a transform
+   * @param   {String} key      KMS encryption key url (or actual key?)
+   * @param   {Object} object An instance of a Webex Teams reaction2Summary object
+   * these objects are returned by various conversation APIs and over mercury
+   * represents an aggregated summary of all reactions to a specific activity
+   * @returns {Promise}         Returns a ctx.transform promise
+   */
+  decryptReaction2summary(ctx, key, object) {
+    if (!object.reactions) {
```

Comment:
should i pull out the body of `decryptReaction2summary` and `decryptReaction2selfsummary` into a helper function, since they are identical?

Reply:
I think they are small enough that there is no need to pull it out, honestly.